.github/dependabot.yml

```
@@ -1,11 +1,26 @@
 ---
 version: 2
+enable-beta-ecosystems: true
 updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
       interval: "weekly"
     rebase-strategy: "disabled"
+    groups:
+      golang-dependencies:
+        patterns:
+          - "github.com/golang*"
```

```suggestion
          - "github.com/golang**"
```
question, will this be `**` or `*` in patterns?

---

it will be `*` as  i have seen [here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups)